[8.9] Adds CloudFormation deployment method to CSPM docs (#3599)
elastic/security-docs: docs/cloud-native-security/cspm-get-started.asciidoc

[discrete]
[[cspm-set-up-cloud-access-section]]
=== Set up cloud account access
The CSPM integration requires access to AWS’s built-in https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_job-functions.html#jf_security-auditor[`SecurityAudit` IAM policy] in order to discover and evaluate resources in your cloud account. There are several ways to provide access.

For most users, the simplest option is to use AWS CloudFormation to automatically provision the necessary resources and permissions in your AWS account. Several manual options are also described below.

[discrete]
[[cspm-set-up-cloudformation]]
=== CloudFormation (recommended)
. Name your integration.
. Under *Setup Access*, select CloudFormation.

> . In a new browser tab, log in as an admin to the AWS account you want to onboard.
> . Return to your Kibana tab. Click *Save and continue* at the bottom of the page.

Reviewer: It probably doesn't have to be a _tab_ -- could be a separate window?
```suggestion
. In a new browser tab or window, log in as an admin to the AWS account you want to onboard.
. Return to {kib}. Click *Save and continue* at the bottom of the page.
```

. Review the information modal, then click *Launch CloudFormation*.

> . A CloudFormation template appears in a new tab. You don't need to modify its configuration.

Reviewer: Is this a browser tab too, or a tab within the UI?

- If `Shared Credential File` is empty, the default directory will be used.
  - For Linux or Unix, the shared credentials file is located at `~/.aws/credentials`.

After providing credentials, <<cspm-finish-manual, finish manual setup>>.

[discrete]
[[cspm-use-iam-arn]]
==== Option 5 - IAM role Amazon Resource Name (ARN)
An IAM role Amazon Resource Name (ARN) is an IAM identity that you can create in your AWS account. You define the role's permissions. Roles do not have standard long-term credentials such as passwords or access keys. Instead, when you assume a role, it provides temporary security credentials for your session.

> To use an IAM role ARN, select `Assume role` under *Preferred manual method*, enter the ARN, and continue to Finish manual setup.

Reviewer: Bold for most things in the UI that the user selects:
```suggestion
To use an IAM role ARN, select *Assume role* under *Preferred manual method*, enter the ARN, and continue to Finish manual setup.
```
